ddtrace/tracer/spancontext_test.go

```diff
@@ -42,7 +42,7 @@ func TestNewSpanContextPushError(t *testing.T) {
 	child.context = newSpanContext(child, parent.context)
 
 	log.Flush()
-	assert.Contains(t, tp.Lines()[1], "ERROR: trace buffer full (2)")
+	assert.Contains(t, tp.Lines()[0], "ERROR: trace buffer full (2)")
```

Comment:
The values were initially set to 0. My previous PR had changes that set them to 1 and I neglected to test this before pushing.